js/core/pci/scan.js

```diff
@@ -802,28 +803,8 @@ pciManager.each(function(pciDevice) {
   // });
 });
 
-// Print PCI devices debug info
-pciManager.each(function(pciDevice) {
-  var address = pciDevice.address();
-  var vector = pciDevice.getIRQVector();
-  var classData = pciDevice.classData();
-
-  var devicePin = 0;
-  if (!pciDevice.isBridge()) {
-    devicePin = pciDevice.interruptPin();
-  }
-
-  var pins = ['dont use', 'A', 'B', 'C', 'D'];
-
-  var info = address.bus.toString(16) + ':' + address.slot.toString(16) + '.' + address.func + ' ' +
-    pciDevice.vendorId().toString(16) + ':' + pciDevice.deviceId().toString(16) + ' ' +
-    classData.className + ' IRQ: ' + vector + ' PIN: ' + pins[devicePin];
-  debug(info);
-});
-
```

Comment:
It'd list all the PCI devices in debug mode, and then when it got to the listPCIDevices function (no matter whether it was in debug or normal) it would just hang.